### PR TITLE
Updated the links to Darwin documents

### PR DESCRIPTION
**Pull request by manukumar6**
## Description
Updated the links to Darwin documents

## Motivation and Context
Darwin release updates

## How Has This Been Tested?
Documentation changes only

## Screenshots (if appropriate)
N/A

## Types of changes
- Documentation updates 

## Checklist

- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes if appropriate.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: products/compute/api/22-12/stable-endpoints.md [hunk 1]
```diff
@@ -38,7 +38,7 @@ Where:
 As a best practice, update your scripts to use the version-specific API endpoints to ensure that your implementation is fully supported.
 For the version-specific APIs, you will have access to the API Reference and Release Notes documentation for changes or updates that may impact you.
 
-When using the version-specific endpoints, you will need to update your automation scripts approximately once-a- year to stay in sync with the product [support lifecycle](https://docs.paloaltonetworks.com/prisma/prisma-cloud/prisma-cloud-admin-compute/upgrade/support_lifecycle.html).
+When using the version-specific endpoints, you will need to update your automation scripts approximately once a year to stay in sync with the product [Support lifecycle for connected components](https://docs.prismacloud.io/en/classic/compute-admin-guide/upgrade/support-lifecycle). If you are upgraded to Darwin, see [Support Lifecycle for Connected Components](https://docs.prismacloud.io/en/enterprise-edition/content-collections/runtime-security/upgrade/support-lifecycle).
```
Comment (Pubs-MV) [inline products/compute/api/22-12/stable-endpoints.md:41]: @manukumar6 for the 22-12 PCCE, I think we need to point to 
https://docs.prismacloud.io/en/compute-edition/22-12/admin-guide/upgrade/support-lifecycle

the links you shared are for PCEE, where we have the notion of classic and Darwin.
The link I have suggested is appropriate if this .md page is for Compute Edition v 22.12.

File: products/compute/api/stable-endpoints.md [hunk 2]
```diff
@@ -40,7 +40,7 @@ Where:
 As a best practice, update your scripts to use the version-specific API endpoints to ensure that your implementation is fully supported.
 For the version-specific APIs, you will have access to the API Reference and Release Notes documentation for changes or updates that may impact you.
 
-When using the version-specific endpoints, you will need to update your automation scripts approximately once-a- year to stay in sync with the product [support lifecycle](https://docs.paloaltonetworks.com/prisma/prisma-cloud/prisma-cloud-admin-compute/upgrade/support_lifecycle.html).
+When using the version-specific endpoints, you will need to update your automation scripts approximately once a year to stay in sync with the product [Support lifecycle for connected components](https://docs.prismacloud.io/en/classic/compute-admin-guide/upgrade/support-lifecycle). If you are upgraded to Darwin, see [Support Lifecycle for Connected Components](https://docs.prismacloud.io/en/enterprise-edition/content-collections/runtime-security/upgrade/support-lifecycle).
```
Comment (Pubs-MV) [inline products/compute/api/stable-endpoints.md:43]: Here what you have look accurate nased on the path of the .md file.

File: products/compute/api/22-12/stable-endpoints.md (hunk 1 above)
Comment (Pubs-MV) [reply products/compute/api/22-12/stable-endpoints.md:41]: this is the case for all URLs with /compute/api/version in the file path.

You have correctly identified the URLs for PCEE on App Sec and Compute. Thanks!

**Review by Pubs-MV — COMMENTED** [review]
Please see comments for PCCE vs PCEE

**Comment by manukumar6** [comment]
Thanks @Pubs-MV. I've fixed the links for the Compute pages.

File: products/compute/api/stable-endpoints.md (hunk 2 above)
Comment (manukumar6) [reply products/compute/api/stable-endpoints.md:43]: This one is the CWPP/PCEE page: https://pan-dev-f1b58--pr532-njvh2q87.web.app/prisma-cloud/api/cwpp/stable-endpoints/ and I've kept it as-is.

File: products/compute/api/22-12/stable-endpoints.md (hunk 1 above)
Comment (manukumar6) [reply products/compute/api/22-12/stable-endpoints.md]: You're right. These are all Compute/PCCE page. I've updated the links as suggested.

**Review by Pubs-MV — APPROVED** [review]

**Pull request merged**